portfolio/src/main/webapp/script.js

```diff
@@ -77,3 +77,100 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+/** Places I have lived. */
+var placesLived = [
```

Comment:
[Nit] Don't use 'var' use 'let' instead (or 'const' for constants).  'var' is global scope and can easily lead to bugs.

Same below.